### PR TITLE
Fix Dependabot alert

### PR DESCRIPTION
**Pull request by nguyenhoangvu1803**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
     "require":{
         "shopware/core": "~6.4.6",
         "shopware/storefront":"*",
-        "phpunit/phpunit": "^9.5"
+        "phpunit/phpunit": "^9.5",
```
Comment (ChristopherDosin) [inline composer.json:18]: shouldn't it be part of `require-dev` ?

Comment (nguyenhoangvu1803) [reply composer.json:18]: Your mean is phpunit or dompdf :D

Comment (ChristopherDosin) [reply composer.json:18]: What do you think as a dev - Should a testing framework like phpunit be part of `require` or is it only required for dev?

Comment (nguyenhoangvu1803) [reply composer.json:18]: Ah ok, I got it

Comment (ChristopherDosin) [reply composer.json]: ok hehe :D 🥇

**Review by hungmac-sw — APPROVED** [review]

**Pull request merged**